integration/testdata/node_app/node_app.csproj

```diff
@@ -31,13 +30,12 @@
     </Exec>
     <Error Condition="'$(ErrorCode)' != '0'" Text="Node.js is required to build and run this project. To continue, please install Node.js from https://nodejs.org/, and then restart your command prompt or IDE." />
     <Message Importance="high" Text="Restoring dependencies using 'npm'. This may take several minutes..." />
-    <Exec WorkingDirectory="$(SpaRoot)" Command="npm --dev install" />
+    <Exec WorkingDirectory="$(SpaRoot)" Command="npm install" />
   </Target>
 
   <Target Name="PublishRunWebpack" AfterTargets="ComputeFilesToPublish">
     <!-- As part of publishing, ensure the JS resources are freshly built in production mode -->
-    <Exec WorkingDirectory="$(SpaRoot)" Command="export PATH=$PATH:$(SpaRoot)/node_modules/.bin" />
-    <Exec WorkingDirectory="$(SpaRoot)" Command="npm --dev install" />
+    <Exec WorkingDirectory="$(SpaRoot)" Command="npm install" />

```

Comment:
My only quick question is did you change this part file by hand or did you have this auto-generated? If you did change it by hand why did you make this chage?

Reply:
I regenerated the app completely with `dotnet new`. I wonder if that's why I had to `pack build` with `NODE_ENV`

Reply:
gonna test it out

Reply:
woot it worked @ForestEckhardt, readded in the latest commit.